pkg/minikube/machine/cache_images.go

```diff
@@ -106,24 +108,26 @@ func LoadImages(cc *config.MachineConfig, runner command.Runner, images []string
 		return errors.Wrap(err, "runtime")
 	}
 
+	cli, err := client.NewEnvClient() // docker cli for image id
+
 	for _, image := range images {
 		image := image
 		g.Go(func() error {
-			ref, err := name.ParseReference(image, name.WeakValidation)
 			if err != nil {
-				return errors.Wrap(err, "image name reference")
+				glog.Infof("error retrieving oci daemon client %v", err)
+				return err
 			}
 
-			img, err := retrieveImage(ref)
+			ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
+			defer cancel()
+			img, _, err := cli.ImageInspectWithRaw(ctx, image)
 			if err != nil {
-				return errors.Wrap(err, "fetching image")
+				glog.Infof("error getting image id %v", err)
```

Comment:
This previously resulted in an error being returned. Should this be a higher priority than info? It seems at least warning-level, if not error-level.